cli/dts/lib.deno.ns.d.ts

```diff
@@ -1954,6 +1954,8 @@ declare namespace Deno {
     iterableLimit?: number;
     /** Show a Proxy's target and handler. Defaults to false. */
     showProxy?: boolean;
+    /** Stylize output with ANSI colors. Defaults to false. */
```

Comment:
I know we are totally inconsistent on this, but it would be nice to alphabetise properties when we have a chance instead of leaving it to some sort of arbitrary ordering.